_posts/2021-03-29-Arts and Sciences Recipe Page.md

```diff
@@ -0,0 +1,145 @@
+
+### What textual structure seems appropriate? Consider the hierarchy of your tags.
+
+On a broad organizational level, the document was separated as a recipe list, with a number, title, and (sometimes) multiple paragraphs of writing in each recipe. In addition, there were page numbers and breaks that frequently appeared in the middle of recipes (but not paragraphs, since that doesn't really make sense).
+
+To account for this organization, I broadly formatted my text with paragraph tags, page tags, recipe number tags, and heading tags nested directly inside div tags (with type = "recipe"). Margin text occasionally appeared alongside title content outside of paragraphs, so there are margin tags nested directly inside div tags as well. Also, some lines or double lines appear between titles and paragraphs, so these tags are also nested directly inside div tags. (double lines also appear outside of recipes, so these are the only tag put outside of div tags).
+
+In terms of finer textual structure, there was some tension about  hierarchy when physical aspects of the texts. For example, there were chunks of text including headings/titles, recipe numbers, lines, and margin writing that were all a different ink color (red or brown instead of black). In these cases, I chose to put color tags as the children of other writing or physical tags (aside from the line or double line tags since these are marks themselves rather than indicators of a physical property of enclosed writing).
+
+The finest textual elements were writing aspects which (aside from the 2 document-specific tags) could be reasonably marked as the child of any non-writing aspect tag since they were small in size and clearly contained within other tags. For example, material tags such as that for Oil of Turpentine clearly should be the child of broader tags such as those for paragraphs. Hierarchy between these writing tags was also easily decided by size; for Neopolitan Ointment, for example, the Neopolitan place descriptor is contained within the material tag for Neopolitan Ointment as a whole.  
+
+### What questions about markup emerged throughout the process?
+
+I primarily ran into a lot of hierarchy and how-much-to-tag, but these aren't really questions about markup so much as personal preference and typical practice.
+
+One question I did have was about the use of "type = ____" in the beginning of tags. I only found it used for the div and language tags  in sample code. It can't be that this feature only usable for specific tags, so is this just a coincidence or is it a broader phenomenon that typing tags is uncommon and used only in a few specific cases. What are some other examples of good uses of this type parameter? I experimented in my own code by adding a type parameter to abbreviation tags to store the full word but I'm not sure if this is a reasonable use.
```

Comment:
This is a really good question and shows that you are thinking through the possibilites of XML. The cool thing about using XML to structure a text is that it doesn't have built-in rules about when you can use an ID like HTML does. Whereas in HTML (if you're familiar with it), you use an id like so `<div id="note">` so that you can link up your CSS specifically to that div by writing a specific style under `#note`, in XML, all of the "attributes" (which are subdivisions within tags) are interpreted according to whatever schema and hierarchy you encode into your XSLT program. In other words, the structure is infinitely customizable as long as your hierarchy is consistent. [This tutorial](https://www.w3schools.com/xml/xml_attributes.asp) on XML attributes from W3School explains it pretty clearly. The sky is the limit! I didn't go into tags versus attributes because I was just introducing XML for the first time, but absolutely, if we were building an XML schema for a class project we would want to make use of attributes within certain categories to help refine our data.